osu.Game.Rulesets.Catch/UI/Catcher.cs

```diff
@@ -430,10 +433,10 @@ private void clearPlate(DroppedObjectAnimation animation)
 
         private void removeFromPlate(CaughtObject caughtObject, DroppedObjectAnimation animation)
         {
-            var droppedObject = getDroppedObject(caughtObject);
```

Comment:
Why was this line moved? Is there meaning behind it? If it's required, it's very confusing as to why, to the point I question the rest of the change.

Reply:
It's the same logic as with `clearPlate`. It definitely isn't required, as you won't really notice the optimization for a single object.

Reply:
Right, fair enough.